### PR TITLE
fix: stop using ServiceNodeExclusion feature gate after it is GA

### PR DESCRIPTION
**Pull request by chewong**
Signed-off-by: Ernest Wong <chuwon@microsoft.com>

**Reason for Change**:

Fixes test failure in https://testgrid.k8s.io/sig-windows-master-release#aks-engine-windows-containerd-master

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [x] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [x] No
- [ ] Yes

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [x] includes documentation
- [x] adds unit tests
- [x] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: pkg/api/defaults-controller-manager.go [hunk 1]
```diff
@@ -81,7 +82,10 @@ func (cs *ContainerService) setControllerManagerConfig() {
 	}
 
 	// Enables Node Exclusion from Services (toggled on agent nodes by the alpha.service-controller.kubernetes.io/exclude-balancer label).
-	addDefaultFeatureGates(o.KubernetesConfig.ControllerManagerConfig, o.OrchestratorVersion, "1.9.0", "ServiceNodeExclusion=true")
+	// ServiceNodeExclusion feature gate is GA in 1.19, removed in 1.22 (xref: https://github.com/kubernetes/kubernetes/pull/100776)
+	if !common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.19.0") {
```
Comment (jsturtevant) [inline pkg/api/defaults-controller-manager.go:86]: nit: little awkward to read but looks like we don't have a LessThan function

Comment (jackfrancis) [reply pkg/api/defaults-controller-manager.go:86]: yeah this is the common pattern for less than

Comment (jackfrancis) [reply pkg/api/defaults-controller-manager.go:86]: /lgtm

**Review by mboersma — APPROVED** [review]
/lgtm++

**Pull request merged**